packages/amplify_datastore_plugin_interface/lib/src/types/conflict_handler/datastore_conflict_handler.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';
+import 'package:flutter/foundation.dart';
+
+typedef DataStoreConflictHandler = ConflictResolutionDecision Function(
+    ConflictData);
+
+class ConflictData {
+  Model local;
+  Model remote;
+
+  ConflictData(this.local, this.remote);
+
+  ConflictData.fromJson(ModelType modelType, Map<String, dynamic> localJson,
+      Map<String, dynamic> remoteJson)
+      : local = modelType.fromJson(
+            new Map<String, dynamic>.from(localJson["serializedData"])),
+        remote = modelType.fromJson(
+            new Map<String, dynamic>.from(remoteJson["serializedData"]));
```

Comment:
`new` was deprecated in Dart 2.0

```suggestion
      : local = modelType.fromJson(
            (localJson["serializedData"] as Map).cast<String, dynamic()),
        remote = modelType.fromJson(
            (remoteJson["serializedData"] as Map).cast<String, dynamic());
```